model/ftn/wminitmd.ftn

```diff
@@ -431,8 +432,12 @@
       CHARACTER(LEN=6)        :: ACTION(11), YESXX, XXXNO
       CHARACTER(LEN=8)        :: LFILE, STTIME
 !/SHRD      CHARACTER(LEN=9)        :: TFILE
-      CHARACTER(LEN=13)       :: STDATE, MN, TNAMES(7)
-      CHARACTER(LEN=40)       :: PN 
+      CHARACTER(LEN=13)       :: STDATE, MN, TNAMES(9)
+      CHARACTER(LEN=40)       :: PN
+      CHARACTER(LEN=13),                                              &
+                  ALLOCATABLE :: INAMES(:,:), MNAMES(:)
+      CHARACTER(LEN=40),                                              &
+                  ALLOCATABLE :: PNAMES(:)
```

Comment:
I think something is not quite right here: INAMES, MNAMES and PNAMES are defined twice...

Reply:
I suggest we just delete the lines you added @ukmo-jianguo-li . It looks like you just change the formatting so they are not split over multiple lines, I think...